README.md

```diff
@@ -1,7 +1,138 @@
-# OPFSampler
+# OPFSampler 
 
 [![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://invenia.github.io/OPFSampler.jl/stable)
 [![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://invenia.github.io/OPFSampler.jl/dev)
 [![Build Status](https://travis-ci.com/invenia/OPFSampler.jl.svg?branch=master)](https://travis-ci.com/invenia/OPFSampler.jl)
 [![Build Status](https://ci.appveyor.com/api/projects/status/github/invenia/OPFSampler.jl?svg=true)](https://ci.appveyor.com/project/invenia/OPFSampler-jl)
 [![Codecov](https://codecov.io/gh/invenia/OPFSampler.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/invenia/OPFSampler.jl)
+
+# Goal
+The goal of this package is to provide functions that take a power grid as input, vary its parameters and generate feasible DC and AC-OPF samples along with the OPF solution. This helps the user to explore a variety of distinct active sets of constraints for the synthetic cases and mimic the time-varying behaviour of the OPF input parameters.
+
+## Sampler
+`RunDCSampler()` and `RunACSampler()` are the two main functions in this section that trigger the sampler to generate DC and AC-OPF samples, respectively.
```

Comment:
Add that we are using PowerModels somewhere, or just resolve if you already have (i didn't see it).

Reply:
Similarly, pg-lib. Both are implied, but the latter is probably worth mentioning explicitly because that's where we're populating the initial grid data from, but compared to powermodels, it doesn't have a dependency. (I thnk).

Reply:
[Note, re: pg-lib, it doesn't need to be a dependency, but make it easier for a user to get up an running by adding this in]

Reply:
that makes sense. I have added pglib-opf but will put a link on it as well and also would mention powermodels that is not mentioned atm.